examples/5.nlp/intent_classification/intent_classification.ipynb

```diff
@@ -11,11 +11,22 @@
   },
```

Comment:
<p>The purpose of this and other outputs of labeled data was to let the user of the notebook know how their results should look like. If you believe that it is not necessary please remove it from other notebooks as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/83/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>